src/en/guide/conf/config/logging.adoc

```diff
@@ -1,4 +1,4 @@
-Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback.xml` file.
+Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback-spring.xml` file.
```

Comment:
Should we mention that `logback.xml` still works?
For example: '... and can be configured with the `grails-app/conf/logback-spring.xml` or `grails-app/conf/logback.xml` files.'

Reply:
If we are moving to `logback-spring.xml` my opinion is not to document it. `logback.xml` will work for applications that are ported to Grails 7, but I would not make it an "official" feature documenting it.

Reply:
I've documented `logback-spring.xml` with a link to the Spring documentation for all the other options. Let me know if that convinces you

Reply:
Could we add a `NOTE:` that `logback.xml` still works but `logback-spring.xml` is now recommended (maybe linking to https://docs.spring.io/spring-boot/reference/features/logging.html#features.logging.custom-log-configuration) then I think we could drop all other mentions of `logback.xml` in the docs.